.circleci/config.yml

```diff
@@ -198,6 +218,23 @@ commands:
       - restore_cache:
           keys:
           - v1.2.5-deps-{{ checksum "get_deps.sh" }}-gpu
+      - restore_cache:
+          keys:
+            - v1.2-tests_data-gpu
```

Comment:
what is the reason we are having an additional cache key? Should be the same for both modes, no?

Reply:
You're right, that's how it was before, but I got `access denied` when restoring cache, so I tried this option instead....